src/axom/core/tests/core_array.hpp

```diff
@@ -11,21 +11,18 @@
 
 #include <algorithm>
 
-namespace axom
-{
-namespace internal
+namespace
```

Comment:
Nearly all the changes in this file are due to removing the utility functions from the `axom` and `axom::internal` namespace.

Reply:
Why did you remove the utility functions from `axom` and `axom::internal`, especially if they are only in the tests?

Reply:
Judgement call, I guess. It was part of my debugging process and deciphering why the test was failing. 

I felt that including the code in this file in the `axom` namespace was obscuring which parts were from the main `axom/core` vs. which were provided only in this test file. Also, since `Array` is a common class name, marking it explicitly as `axom::Array` seemed (to me) to better highlight that we were using an axom class.